requirements/base.txt

```diff
@@ -1,16 +1,16 @@
 #
-# This file is autogenerated by pip-compile with python 3.8
+# This file is autogenerated by pip-compile with python 3.10
```

Comment:
Why are we running with Python 3.10?  Don't we try to keep all of our environments in sync on the Python version?

Reply:
Ah, good point, made a new venv with the wrong version, I'll update.